### PR TITLE
Updating Oculus API to 1.41 and adding eye height and boundary options

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
## Overview

Pr to address issues on the Oculus platform

## Changes:

- [x] Updating API to V 1.41
- [x] Porting more functions from the Oculus API for eye height and boundary
- [x] Teleporting working as expected

## Postponed to another PR
- [x] Implement Boundary system

### CODE REVIEW

File: XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs [hunk 1]
```diff
@@ -95,88 +95,158 @@ internal static OVRControllerBase[] Controllers
         internal static Controller activeControllerType = Controller.None;
         internal static Controller connectedControllerTypes = Controller.None;
 
+        public static bool Initialized
+        {
+            get
+            {
+                return ovrp_GetInitialized() == Bool.True;
```
Comment (StephenHodgson) [inline XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs]: These could use the cleaner syntax to help reduce the number of lines. (Same with all the things in this class)

```
public static bool Initializeed
{
    get => ovrp_GetInitialized() == Bool.True;
}
```

File: XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs [hunk 2]
```diff
@@ -95,88 +95,158 @@ internal static OVRControllerBase[] Controllers
...
+            }
+        }
+
+        public static float EyeDepth
+        {
+            get
+            {
+                if (!Initialized)
+                    return 0.0f;
```
Comment (StephenHodgson) [inline XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs]: missing braces

File: XRTK.Oculus/Packages/com.xrtk.oculus/package.json [hunk 3]
```diff
@@ -15,6 +15,6 @@
   "src": "Assets/XRTK.Oculus",
   "author": "XRTK Team (https://github.com/XRTK)",
   "dependencies": {
-    "com.xrtk.core": "0.1.21"
+    "com.xrtk.core": "0.1.25"
```
Comment (StephenHodgson) [inline XRTK.Oculus/Packages/com.xrtk.oculus/package.json]: Don't forget to bump the package release version as well

Comment (SimonDarksideJ) [reply XRTK.Oculus/Packages/com.xrtk.oculus/package.json]: Will double-check, I believe you upped the version in a second PR

File: XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs (hunk 2 above)
Comment (SimonDarksideJ) [reply XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs]: Oculus API code, can't change to retain license

File: XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs (hunk 1 above)
Comment (SimonDarksideJ) [reply XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs]: Oculus API code, can't change to retain license

File: XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs (hunk 2 above)
Comment (StephenHodgson) [reply XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs]: When why is this code checked into the project?
Shouldn't we just reference a submodule?

**Pull request merged**